app/tests/spec/views/complete_reset_password.js

```diff
@@ -49,6 +51,7 @@ define(function (require, exports, module) {
 
     function initView() {
       view = new View({
+        able: able,
```

Comment:
This was added so that unit tests pass on Chrome. Tracing it back, Chrome triggers `onPasswordBlur`  in the `complete_reset_password` view where as Firefox does not, ref https://github.com/mozilla/fxa-content-server/blob/master/app/scripts/views/mixins/password-prompt-mixin.js#L79. `onPasswordBlur` uses able to get a couple different properties. Do you think it is better to leave this out? In either case, I can add a quick blurp about it.